src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -26,19 +28,48 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+const usePrevious = (value) => {
+  const ref = React.useRef(null);
+  React.useEffect(() => {
+    // If data is loaded, store the id of the first post to ref
+    if (value !== undefined) {
+      ref.current = value[0][0].id;
+    }
+  });
+  return ref.current;
+};
+
 const REFRESH_INTERVAL = 5 * 60 * 1000; /* refresh data every 5 minutes */
 
 const Posts = () => {
   const classes = useStyles();
+  // Call setBadge(true) to show the favicon badge when the page is not visible
+  const setBadge = useFaviconBadge(false); // pass true here to badge from the start
+  const isVisible = usePageLifecycle();
   const { telescopeUrl } = useSiteMetaData();
   const { data, size, setSize, error } = useSWRInfinite(
     (index) => `${telescopeUrl}/posts?page=${index + 1}`,
     (url) => fetch(url).then((r) => r.json()),
     {
       refreshInterval: REFRESH_INTERVAL,
+      refreshWhenHidden: true,
     }
   );
 
+  // Set to true to show badge, false otherwise.  Badge only shows when not visible.
+  const oldData = usePrevious(data);
+  React.useEffect(() => {
+    if (oldData) {
+      const newData = data[0][0].id;
+      if (oldData !== newData) {
+        setBadge(true);
+      }
+      if (isVisible) {
```

Comment:
This should be happening in the `useFaviconBadge` hook, we don't need to expose it here.  You should just set it to false, and let the hook figure out when to show/hide it.